core/src/main/resources/core-reference.conf

```diff
@@ -43,3 +43,60 @@ crossdata-core.catalog.caseSensitive = true
 #crossdata-core.catalog.zookeeper.retryAttempts = 5
 #crossdata-core.catalog.zookeeper.retryInterval = 10000
 
+
+
+#####################################
+#                                   #
+#      Streaming configuration      #
+#                                   #
+#####################################
+
+####### StreamingCatalog ###########
+crossdata-core.streaming.catalog.class="org.apache.spark.sql.crossdata.catalog.ZookeeperStreamingCatalog"
+crossdata-core.streaming.catalog.zookeeper.connectionString = "localhost:2181"
+crossdata-core.streaming.catalog.zookeeper.connectionTimeout = 15000
+crossdata-core.streaming.catalog.zookeeper.sessionTimeout = 60000
+crossdata-core.streaming.catalog.zookeeper.retryAttempts = 5
+crossdata-core.streaming.catalog.zookeeper.retryInterval = 10000
+
+
+
+####### KafkaReceiverOptions ###########
+crossdata-core.streaming.receiver.kafka.connection = "localhost:2181:9092"
+#crossdata-core.streaming.receiver.kafka.numPartitions = "1"
+crossdata-core.streaming.receiver.storageLevel = "MEMORY_AND_DISK_SER"
+#crossdata-core.streaming.receiver.kafka.options.someKey = "someValue"
+
+####### StreamingOptions ###########
+crossdata-core.streaming.atomicWindow = "5"
+crossdata-core.streaming.maxWindow= "10"
+crossdata-core.streaming.outputFormat = "JSON"
```

Comment:
Is there any place where we have the possible output formats? If not, at least they should be specified here with a comment or in the Confluence doc